__tests__/app.test.js

```diff
@@ -132,3 +132,56 @@ describe("GET /api/users", () => {
     });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  describe("api calls", () => {
+    test("200: updates votes property by the specified amount + responds with the updated article", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.votes).toEqual(101);
```

Comment:
We want the whole of the article to be returned from this endpoint - so you need to check for all keys expected either by using a .toEqual or individually. Remember - we know the values for article with article_id of 1, so check for specific values not just expect.any(String) for example.